src/components/AgentRow.tsx

```diff
@@ -13,19 +15,67 @@ const AgentRow: React.FC<AgentRowProps> = (
 ): JSX.Element => {
   const { agent } = props;
 
+  const [showDisplayStates, setShowDisplayStates] = React.useState(false);
```

Comment:
in the website all mention of "displayState" is confined to the 'state'. But in the UI components, the language is `parent/child`. The idea of display states comes from our data structure and the semantics of how the trajectories are organized. But I think at this level in the code, `showChildren` and `hasChildren` is a lot easier to understand

Reply:
I agree it’s easier to understand in this context and I’m willing to change this! I also found it confusing that this same data has a few different names throughout different levels of the code so I thought we could reduce effort in understanding how parts of the code base connect by using the same nomenclature in more places.